src/index.html

```diff
@@ -9,6 +9,11 @@
   <link rel="stylesheet" href="./styles/main.scss">
 </head>
 <body>
-  <h1>Stop watch</h1>
+<div class="watch">
+  <div class="watch__arrow watch__first--sec"></div>
+  <div class="watch__arrow watch__first--min"></div>
```

Comment:
Use modifier for correct classes
```suggestion
  <div class="watch__arrow watch__arrow--sec"></div>
  <div class="watch__arrow watch__arrow--min"></div>
```

Reply:
done